sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -786,6 +794,44 @@ class EthService(
         // apply requested transaction with tracing enabled
         tagStateView.applyTransaction(requestedTx, previousTransactions.length, gasPool, blockContext)
 
+        // return the tracer result from the evm
+        if(blockContext.getEvmResult != null) {
+          blockContext.getEvmResult.tracerResult
+        } else Unit
+      }
+    }
+  }
+
+  @RpcMethod("debug_traceCall")
+  @RpcOptionalParameters(1)
+  def traceCall(params: TransactionArgs, tag: String, config: TraceOptions): Any = {
+
+    applyOnAccountView { nodeView =>
+      // get block to trace
+      val (block, blockInfo) = getBlockById(nodeView, getBlockIdByHashOrTag(nodeView, tag))
+
+      // get state at selected block
+      getStateViewAtTag(nodeView, if(tag=="pending") "pending" else (blockInfo.height).toString) { (tagStateView, blockContext) =>
+
+        // use default trace params if none are given
+        blockContext.setTraceParams(if (config == null) new TraceOptions() else config)
```

Comment:
Enabling tracing here will also enable it for the transactions of the block - but probably doesn't matter, see the other comment.

Reply:
yes, removing the code block reported in the comment above the trace params will be applied only to the message create with the input parameters.